app/controllers/two-factor-auth-controller/get-configure-controller.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+// NPM dependencies
+const logger = require('winston')
+const qrcode = require('qrcode')
+
+// Local dependencies
+const {response} = require('../../utils/response.js')
+const paths = require('../../paths')
+
+const PAGE_PARAMS = {
```

Comment:
this needs to be declared inside the exported function as you are assigning to it. Currently this object is shared between all requests to this controller.